recording_server/src/media_context_macos.ts

```
@@ -0,0 +1,294 @@
+            })
+        })
+        return this.process
+    }
+
+    protected async stop_process() {
+        if (!this.process) {
+            console.warn('Already stopped')
+            return
+        }
+
+        let res = this.process.kill('SIGTERM')
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'res' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)